icons/sqlite/sqlite-plain.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="180"
```

Comment:
I think the width, height, and viewbox should be `128`? 🤔
If you're having trouble with resizing, try to use [**this tool**](https://www.iloveimg.com/resize-image/resize-svg).